src/test/ui/proc-macro/span-api-tests.rs

```diff
@@ -11,7 +11,7 @@ extern crate span_test_macros;
 
 extern crate span_api_tests;
 
-use span_api_tests::{reemit, assert_fake_source_file, assert_source_file, macro_stringify};
+use span_api_tests::{reemit, assert_source_file, macro_stringify};
```

Comment:
According to https://github.com/rust-lang/rust/pull/66364#issuecomment-590095860, this is likely the only thing we can do for now.

Can you open an issue about perhaps testing the case `assert_fake_source_file` was used to test (`SourceFile::is_real()` returning `false`) and leaving a FIXME comment here mentioning that issue and/or in the `auxiliary` file where `assert_fake_source_file` is defined?

Maybe it doesn't matter that much, but I wanted to come back to this before I forget again about it.

Reply:
Left a FIXME: https://github.com/rust-lang/rust/issues/69775.